src/DGtal/kernel/sets/DigitalSetFromMap.h

```diff
@@ -269,6 +269,13 @@ namespace DGtal
      */
     Iterator end();
 
+    /**
+     * Give access to the underlying container.
+     * @return a (might be const) reference to the stored container.
+    */
+    inline const Image & container() const { return *myImgPtr;};
```

Comment:
Should we add a typedef Image Container; ?

Reply:
Added